packages/office-ui-fabric-react/src/components/Spinner/Spinner.tsx

```diff
@@ -14,7 +14,9 @@ export class Spinner extends React.Component<ISpinnerProps, any> {
     return (
       <div className={ css('ms-Spinner', className) }>
         <div className={ css('ms-Spinner-circle',
-          { 'ms-Spinner--normal': type === SpinnerType.normal },
```

Comment:
to continue to support SpinnerType.normal, we'd need to leave this one in. I see the class is still there, so SpinnerType.normal should still add ms-Spinner--normal until fully removed.